es_utils.py

```diff
@@ -267,3 +267,26 @@ def es_send_external_stats(stats_dict_file_path, opts_dict_file_path, cpu_normal
   sdata["@timestamp"]=file_stamp*1000
   try:send_payload(es_index_name+"-"+week, es_doc_name, index_sha, json.dumps(sdata))
   except Exception as e: print(e.message)
+
+def getExternalsESstats(cmsdist='*', arch='*', lastNdays=30, page_size=0):
+    stats = es_query(index='externals_stats_summary_testindex*',
```

Comment:
@mrodozov , we should now start using production index i.e no need to have _testindex in the name now

Reply:
I'm changing the name in the uploading script, than a re-run should create the renamed indexes and after that erasing the old ones

Reply:
sounds good.